### PR TITLE
notebooks: add two examples

### PR DESCRIPTION
**Pull request by jordanrfrazier**
Adds the examples from https://colab.research.google.com/drive/1vHDf5u8cWKQpecAHuC8iRhIjd_VsavuG?usp=sharing#scrollTo=a6d88d66

And https://colab.research.google.com/drive/1RCZrAbL10MYrikjPvN0Nf4GMua09Dohu?usp=sharing#scrollTo=4uJebWz9JfAu

TODO: 
* Update branch  in links to `main` before merging.

### CODE REVIEW

File: QA_on_AstraDB.ipynb [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+      "id": "cf05e3f2",
+      "metadata": {
+        "id": "cf05e3f2"
+      },
+      "source": [
+        "# Knowledge Base Search on Proprietary Data powered by Astra DB\n",
+        "This demo guides you through setting up RAGStack, using Astra DB with Vector Search, CassIO, and Open AI to implement a generative Q&A over your own documentation.\n",
+        "\n",
+        "Jupyter notebook for generative Q&A for documents is powered by [RAGStack](https://www.datastax.com/products/ragstack) using [Astra Vector Search](https://docs.datastax.com/en/astra-serverless/docs/vector-search/overview.html), [OpenAI](https://github.com/openai/), and [CassIO](https://cassio.org/).\n",
+        "\n",
```
Comment (epinzur) [inline QA_on_AstraDB.ipynb:21]: seems like these two sentences are stating basically the same thing. Maybe drop the second one but copy the links to the first?

File: QA_on_AstraDB.ipynb [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+        "## Demo Summary\n",
+        "ChatGPT excels at answering questions, but only on topics it remembers from its training data. It offers you a nice dialog interface to ask questions and get answers.\n",
+        "\n",
+        "But what do you do when you have your own documents? How can you leverage the GenAI and LLM models to get insights in those?\n",
+        "\n",
+        "Think of an Q/A Bot that you want to provide to your customers for asking questions against the documentation of your products.\n",
+        "\n",
+        "For being able to do so, you have to implement your own ChatGPT-like solution.\n",
+        "The implementation requires\n",
+        "1. Analysing your existing documents and store the information\n",
```
Comment (epinzur) [inline QA_on_AstraDB.ipynb:43]: sp: analyzing 

also change store -> storing

Comment (epinzur) [reply QA_on_AstraDB.ipynb:43]: or change to: Analyze your existing documents and store the information.

File: QA_on_AstraDB.ipynb [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+        "2. Providing search capabilities for your questions to get answers\n",
+        "\n",
+        "This is solved by using LLM models. Ideally you embed the data as vectors and store them in a vector database and then use the LLM models on top of that database.\n",
+        "\n",
+        "This notebook demonstrates a two-step Search-Ask method for enabling GPT to answer questions using a library of reference on your own documentations based on Astra DB vector search.\n",
+        "\n",
+        "\n"
```
Comment (epinzur) [inline QA_on_AstraDB.ipynb:50]: should we say anything about this being a RAG approach?

File: QA_on_AstraDB.ipynb [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "J5VPZclr6n67",
+        "outputId": "4837f954-06a4-430f-dd1f-5ed9c2f891f4"
+      },
+      "outputs": [],
+      "source": [
+        "# Set your secret(s) for LLM access:\n",
+        "# We will use OpenAI embeddings, so please provide your OpenAI AKP Key\n",
```
Comment (epinzur) [inline QA_on_AstraDB.ipynb:317]: i think this should be an API key, not a AKP key.

File: QA_on_AstraDB.ipynb [hunk 5]
```diff
@@ -0,0 +1,577 @@
...
+      "outputs": [],
+      "source": [
+        "#Import the needed libraries and declare the LLM model\n",
+        "import langchain\n",
+        "from langchain.embeddings import OpenAIEmbeddings\n",
+        "from langchain.text_splitter import CharacterTextSplitter\n",
+        "from langchain.vectorstores import Cassandra\n",
+        "from langchain.document_loaders import TextLoader\n",
+        "from langchain.document_loaders import PyPDFLoader\n",
+        "import os\n",
```
Comment (epinzur) [inline QA_on_AstraDB.ipynb:343]: os already imported above

**Review by epinzur — APPROVED** [review]

File: RAGStack_using_cassio.ipynb [hunk 6]
```diff
@@ -0,0 +1,999 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "eZS2Xsy0WY-c",
+      "metadata": {
+        "id": "eZS2Xsy0WY-c"
+      },
+      "source": [
+        "## **Dataset Setup**\n",
```
[the remark this answers is not in the document]
Comment (jordanrfrazier) [reply RAGStack_using_cassio.ipynb]: Ended up using the cassio docs -- that made more sense.

**Pull request merged**